src/main/java/frc/robot/Config.java

```diff
@@ -222,7 +225,7 @@ public static final class PhotonConfig{
       //left and right: between center of robot and aprilTag
```

Comment:
These offsets also need to be tweaked for the competition. Note these offsets are in the field's coordinate frame on the blue alliance and not the tags coordinate frame.